[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-3 (#114275)
openshift/openshift-docs: modules/compliance-unapplying.adoc

[id="compliance-unapplying_{context}"]
= Unapplying a remediation

[role="_abstract"]

> It might be required not to apply a remediation that was previously applied.

Reviewer: ```suggestion
You can unapply a remediation that was previously applied.
```
